lib/util/git/index.ts

```diff
@@ -1176,7 +1176,7 @@ export async function prepareCommit({
         try {
           /* v8 ignore next 2 -- TODO: add test */
           const addParams =
-            fileName === configFileNames[0] ? ['-f', fileName] : fileName;
+            fileName === getConfigFileNames()[0] ? ['-f', fileName] : fileName;
```

Comment:
This seems wrong to me. 

Anyone aware why this does only take `renovate.json` into consideration?